docs/reference/webapp/dataset_list.md

```diff
@@ -1,28 +1,29 @@
-# Workspace
+# Dataset list
 
 ![Screenshot of the Workspace page](../../_static/reference/webapp/workspace1.png)
 
-The _Workspace_ page is mainly a **searchable and sortable list** of **datasets**.
-It is the **entry point** to the Rubrix web app and is composed of the following 3 components.
+The _Dataset list_ page is mainly a **filtrable, searchable and sortable list** of **datasets**.
+It is the **entry point** to the Rubrix web app and is composed of the following three components.
 
 ## Search bar
 
 The "_Search datasets_" bar on the top allows you to search for a specific dataset by its name.
 
 ## Dataset list
```

Comment:
Let's rename this to 'Dataset table', otherwise we have to titles named 'Dataset list'
```suggestion
## Dataset table
```